server/data/data-library/src/main/java/org/apache/james/domainlist/lib/DomainListConfiguration.java

```diff
@@ -93,13 +93,15 @@ public Builder addConfiguredDomains(Domain... domains) {
 
         public DomainListConfiguration build() {
             return new DomainListConfiguration(
-                autoDetectIp.orElse(true),
-                autoDetect.orElse(true),
+                autoDetectIp.orElse(false),
```

Comment:
oh, a breaking change! (:

Reply:
This, is intended! Maybe I should mention it in the changelog?

Reply:
maybe